manifest.json

```diff
@@ -68,6 +68,15 @@
                     "cs": "Mohou se noví uživatelé registrovat?"
                 },
                 "default": false
+            },
+            {
+                
+           "name": "update",
+           "type": "boolean",
+           "ask":{
+                "en": "Making this option to true will give you an option to force upgrade to new version without the package being updated to that version. WARNING: This can be risky and break your app if specific version have additional steps than noraml upgrade. Keep the option false, if you are not sure."
+           },
+           "default": false
             }
```

Comment:
I don't think it's a good idea to let people install a newer version than the last tested... Or we will have the same issue as letting all people going to an unknown version

Reply:
Ok